src/vs/vscode.d.ts

```diff
@@ -6538,11 +6538,28 @@ declare module 'vscode' {
 		getParent?(element: T): ProviderResult<T>;
 	}
 
+	/**
+	 * Label describing the [Tree item](#TreeItem)
+	 */
+	export interface TreeItemLabel {
+
+		/**
+		 * A human-readable string describing the [Tree item](#TreeItem).
+		 */
+		label: string;
+
+		/**
+		 * Ranges in the label to highlight.
+		 */
+		highlights?: { start: number, end: number }[];
```

Comment:
We have been using tuples instead of objects so far, e.g [`valueSelection`](https://github.com/Microsoft/vscode/blob/b431db5e957466c5f0b7725f891f4e8752caf786/src/vs/vscode.d.ts#L1770) and [`ParameterInfo#label`](https://github.com/Microsoft/vscode/blob/b431db5e957466c5f0b7725f891f4e8752caf786/src/vs/vscode.d.ts#L3035) use `[number,number]`